components/gc-intro/gc-intro-block-en.html

```diff
@@ -0,0 +1,88 @@
+---
+title: Introduction block pattern
+description: Documentation and working example for the introduction block pattern
+language: en
+altLangPage: gc-intro-block-fr.html
+dateModified: 2023-08-09
+layout: no-container
+---
+<div class="container">
+	<h1 property="name" id="wb-cont" dir="ltr">{{ page.title }}</h1>
+
+	<span class="wb-prettify all-pre hide"></span>
+	<p>The introduction block pattern introduces the content of a landing page.</p>
+
+	<h2>On this page</h2>
+	<ul>
+		<!-- <li><a href="#inc-components">Included components</a></li> -->
+		<li><a href="#before-start">Before you start</a></li>
+		<li><a href="#accessibility-guidance">Accessibility guidance</a></li>
+		<li><a href="#variations">Variations</a></li>
+		<li><a href="#eval-report">Evaluation and report</a></li>
+		<li><a href="#technical-docs">Technical documentation</a></li>
+	</ul>
+
+	<h2 id="before-start">Before you start</h2>
+	<p>The introduction block pattern is a mandatory element for institutional landing pages and topic pages. It’s optional for other landing pages.</p>
+
+	<ul>
+		<li><a href="">How to set up the institutional landing page</a></li>
+		<li><a href="">Introduction block pattern</a></li>
+	</ul>
+
+	<h2 id="accessibility-guidance">Accessibility guidance</h2>
+	<p>List of accessibility guidances to be followed to ensure component is used in respect of the accessibility guideline and EN 301 549 standard (including WCAG 2.1 Level AA).</p>
+	<p>There is <span>no accessibility guidance</span> defined.</p>
+
+	<h2 id="variations">Variations</h2>
+	<p>The introduction block pattern consists of:</p>
+	<ul>
+		<li>title</li>
+		<li>short description</li>
+	</ul>
+	<p>It can also include:</p>
```

Comment:
```suggestion
	</ul>
	<pre><code>
	&lt;h1 property="name" id="wb-cont" dir="ltr">Introduction block pattern&lt;/h1>
	&lt;p>The introduction block pattern introduces the content of a landing page.&lt;/p>
	</code></pre>
	<p>It can also include:</p>
```

Reply:
If we do the demo on a separate page, it seems we don't need this edit?

Reply:
yes, you are right